playbook/app/pb_kits/playbook/pb_section_separator/docs/_section_separator_vertical.html.erb

```diff
@@ -11,3 +13,41 @@
         Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
     <% end %>
 <% end %>
+</br>
+<div class="clear"></div>
```

Comment:
Just curious: what is a reason for having a div here if there's nothing inside?

Reply:
It's just for the class: scss, what is inside it is in the scss file. 

I needed to create a class with "clear" to clear both sides because when I added a new item underneath or above, they would overlap and not take the code I intended.  If you have a better option how to handle that, I am more than happy to hear. I just learned its an easy way around it-- it might not be the correct way though.

Reply:
Actually figured out a way without the devs, so thanks for the question!